dsnp-wallet/dsnp-wallet/Source/Views/Shared/SharedButton.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  SharedButton.swift
+//  dsnp-wallet
+//
+//  Created by Rigo Carbajal on 5/11/21.
+//
+
+import UIKit
+
+enum SharedButtonStyle {
+    case primary
+}
+
+class SharedButton: UIButton {
```

Comment:
Let's be careful not to copy things from the UsApp if we're not using them. I'm ok with having a SharedButton but it's logic should be kept to a minimal subset of only the logic that is currently being used in the Wallet.